### ISSUE TITLE
NTR: Hepatobiliary scintigraphy: gallbladder filling absent or delayed (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8969]
New term request Hepatobiliary scintigraphy: gallbladder filling absent or delayed (elthat_121120163139): Cholescintigraphy

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000378
- label: Delayed scintigraphic gallbladder filling
- parents: [Abnormal gallbladder physiology (HP:0012438)](https://hpo.jax.org/app/browse/term/HP:0012438)
- pmid: PMID:30855831
- def: Hepatobiliary scintigraphy is a nuclear imaging technique that permits evaluation of the liver and biliary system for acute and chronic cholecystitis, biliary atresia, biliary obstruction, and post-surgical biliary leak. With this investigation, a radiotracer is administered intravenously, bound to albumin, transported to the liver, and excreted into the biliary system. This term refers to a longer than normal time for the radiotracer to be visualized in the gallbladder. This finding may be a sign of chronic gallbladder disease or partial obstruction of the common bile duct.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8969
- synonyms:
